### DISCUSSION TITLE
ADAPTIVE HEX MESH

### DISCUSSION THREAD

**Discussion by CinekPoland** [OpenRadioss/OpenRadioss#949] [Q&A]
Doeas Openradioss allow for adaptive refinement hex elements during simulaton ?

**Comment by MarianBulla** [comment]
Hello,
adaptive mesh is only possible for shells.
Thanks and best regards,
Marian

**Reply by CinekPoland** [reply]
Thank you for your answer  Is there any option to keeping good quality mesh during simulation which include lare deformation and displacement ? for example during forging simulation ?

**Comment by MarianBulla** [comment]
For large deformation, I would try the ALE (or SPH) approach.

**Reply by CinekPoland** [reply]
Yes this is good direction, I will try SPH. It is no problem to use friction phenomena in SPH ?

**Comment by JiancaiZheng** [comment]
Hello,
I want to implement numerical calculations for compressible fluid adaptive meshing based on OpenRadioss. Could you provide a feasible approach?

Jiancai Zheng
